src/pages/resultsView/mutation/ResultsViewMutationMapper.tsx

```diff
@@ -50,6 +50,13 @@ export default class ResultsViewMutationMapper extends MutationMapper<IResultsVi
         }
     }
 
+    protected get isMutationTableDataLoading() {
+        return (this.props.store.clinicalDataForSamples &&
```

Comment:
lets use my new function getMobxPromiseGroupStatus.  Also, why check for existence of promise?

Reply:
good point, I dont know